src/containers/internalContactOverview.js

```diff
@@ -136,43 +324,61 @@ class InternalContactOverview extends PureComponent {
             onLeftIconButtonClick={_=> this.setState({drawer: !this.state.drawer})}
           />
           <Avatar
-            src='https://www.watsonmartin.com/wp-content/uploads/2016/03/default-profile-picture.jpg'
+            src= {this.state.currentUser.picture}
```

Comment:
Are we really going to have a profile picture of a user? 
Or we are just going to add a default picture

Reply:
I removed this avatar now. Think it's not needed

Reply:
as in now is in the code I am working on that is not pushed to github yet 😆